stumpy/fastmath.py

```diff
@@ -0,0 +1,79 @@
+import importlib
+
+from numba import njit
+
+from stumpy import config
```

Comment:
This should be `from . import config`!!

Reply:
Note that we only do this since `fastmath.py` is INSIDE of the STUMPY package (while unit tests are OUTSIDE of the package)

Reply:
Correct! My bad. Thanks for pointing that out 👍

Reply:
I've added a "check" to `test.sh` to catch this:

https://github.com/TDAmeritrade/stumpy/blob/b6d9bcebcc278cccb7b1c131dae5b4ea4d972adc/test.sh#L103-L111

Reply:
Yep, noticed that. Awesome!!